### DISCUSSION TITLE
Return Similarity scores from Linker

### DISCUSSION THREAD

**Discussion by hananshandler** [moj-analytical-services/splink#2569] [Q&A]
I've built a layered entity resolution process where I run a tuned splink model, then run the output with some additional data into a tree based model. I've found it performs incredibly well on my data. However, some of the data that feeds into the tree based model is the same jaro winkler similarity scores that splink calculates.

Currently, I'm just using duckdb to calculate those scores again. Is there a way to output the similarity scores that splink generates during the linker process?

Thanks in advance!

**Comment by RobinL** [comment]
Not really I'm afraid, they're always part of an expression like when jaro(val_l, val_r) > x then n, so they're not saved anywhere.  

You could use probably use the deterministic linking functionality to generate the blocked pairs and then run the similarity functions on the results using SQL
